# --- T2-COPYRIGHT-NOTE-BEGIN ---
# T2 SDE: package/*/inkscape/hotfix-libxml.patch
# Copyright (C) 2024 The T2 SDE Project
# 
# This Copyright note is generated by scripts/Create-CopyPatch,
# more information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License version 2 as used by the T2 SDE.
# --- T2-COPYRIGHT-NOTE-END ---

--- inkscape-1.3.2_2023-11-25_091e20ef0f/src/object/uri.cpp.vanilla	2024-02-12 23:14:44.030000000 +0000
+++ inkscape-1.3.2_2023-11-25_091e20ef0f/src/object/uri.cpp	2024-02-12 23:15:23.843000000 +0000
@@ -13,6 +13,8 @@
 
 #include <cstring>
 
+#include <libxml/xmlmemory.h>
+
 #include <giomm/contenttype.h>
 #include <giomm/file.h>
 #include <glibmm/base64.h>
